db/seeds/dev/board_gameSeed.js

```diff
@@ -0,0 +1,24 @@
+const board_gameData = require('../../data/board_games');
+
+const createBoardgame = async (knex, board) => {
+  const boardId = await knex('board_games').insert({
+    name: board.name,
+    description: board.description || board.description_preview,
+    min_players: board.min_players,
```

Comment:
this is such good use of the `or`, I should implement it as well for the music and podcast table. I am assuming that `description` and `previews` are values returned by your promise.

Reply:
Yeah the data on some of these APIs seems a little inconsistent as far as I have seen. I used the && (logical and) operator  on the movies api to fix some breaking bugs (checking for nested element endpoints where the parent doesn't exist)